scripts/jenkins/qa_devstack.sh

```diff
@@ -107,14 +113,28 @@ function h_setup_extra_disk {
 }
 
 function h_setup_devstack {
-    $zypper in git-core which ca-certificates-mozilla net-tools git-review
-    $zypper in 'group(nogroup)'
+    $zypper in git-review
+    $zypper mr -d SUSE_Factory_Head
```

Comment:
Can we fix that in the image that you are using instead?

Reply:
Not sure I understand the question.  S:F:H was the only place I could find a usable git-review, so I had to `zypper ar` it, but I didn't want it enabled when installing anything else, so I chose to disable after installing git-review.

Reply:
In theory, the 42.3 pool repo should be more compatible to SLE12 - and also it is not moving like Factory does. It has git-review-1.25.0

Maybe you could even do
`$zypper in http://download.opensuse.org/repositories/openSUSE:/Leap:/42.3/standard/noarch/git-review-1.25.0-6.2.noarch.rpm`
without adding the whole repo?

Reply:
Nice idea, thanks Bernhard!